projects/core/src/product/store/reducers/product-search.reducer.ts

```diff
@@ -24,7 +25,7 @@ export function reducer(
       const results = action.payload;
 
       if (state.loading) {
-        const res = action.auxiliary ? { auxResults: results } : { results };
+        const res = action.auxiliary ? { auxResults: results } : results;
```

Comment:
Put this back const res = action.auxiliary ? { auxResults: results } : { results };